components/hero.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { MutableRefObject } from "react";
+import Image from "next/image";
+import mZAR from "../images/mZAR-coin.png";
+import Creative from "../images/mesh-creative.png";
+import { TrustedBySection } from "./trusted-by-section";
+
+export interface HeroProps {
+  // ref: MutableRefObject<HTMLDivElement | null>;
```

Comment:
we can delete the HeroProps since it's not used

Reply:
deleted